CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 CHANGELOG
 =========
 
+1.1.5dev
```

Comment:
Should this still be dev?

Reply:
Yep the release date is next week. We can submit final PR at that time.